README.md

```diff
@@ -1,137 +1,156 @@
-    in width instead or use 'basic' or 'conservative' algorithms instead.
--   **Algorithm**: There are several algorithms available:
-    -   *conservative* Count-min Sketch as described on as described on [Wikipedia](https://en.wikipedia.org/wiki/Count%E2%80%93min_sketch).
-        Uses 4 bytes per cell, storing values up to 2^32 - 1
-    -   *log1024* is a conservative-update probabilistic counter with a value that fits into 2 bytes. Values of this counter
-        up to 2048 are precise and larger values are off by +/- 2%. The maximum value is larger than 2^71
-    -   *log8* is a conservative-update probabilistic counter that fits into just 1 byte. Values of this counter
-        up to 8 are precise and larger values can be off by +/- 30%. The maximum value is larger than 2^33        
-
-Memory
-------
-To calculate memory footprint:
-    ( width * depth * cell_size ) + HLL size
-
-Cell size is
-   - 4B for conservative algorithm
-   - 2B for log1024
-   - 1B for log8
-   
-HLL size is 64 KB
-
-Example:
-    width 2^25 (33,554,432), depth 8, logcons1024 (2B) has 2^(25 + 3 + 1) + 64 KB = 536.9 MB
-    Can be pickled to disk with this exact size.
-
-Performance
------------
-### Testing on Wikipedia set
-We have counted unigrams and bigrams in English Wikipedia dataset to evaluate the counters.
-In each case, we have counted the entire data set into all bounter structures:
+However, unlike `dict` or `Counter`, Bounter can process huge collections where the items would not even fit in RAM. This commonly happens in Machine Learning and NLP, with tasks like **dictionary building** or **collocation detection** that need to estimate counts of billions of items (token ngrams) for their statistical scoring and subsequent filtering.
 
-```python
-    with smart_open('title_tokens.txt.gz') as wiki:
-        for line in wiki:
-            words = line.decode().split('\t')[1].split()
-            counter.update(words)
+Bounter implements approximative algorithms using optimized low-level C structures, to avoid the overhead of Python objects. It lets you specify the maximum amount of RAM you want to use. In the Wikipedia example below, Bounter uses 17x less memory compared to `Counter`.
+
+Bounter is also marginally faster than the built-in `dict` and `Counter`, so wherever you can represent your **items as strings** (both byte-strings and unicode are fine, and Bounter works in both Python2 and Python3), there's no reason not to use Bounter instead.
+
+## Installation
+
+Bounter has no dependencies beyond Python >= 2.7 or Python >= 3.3 and a C compiler:
+
+```bash
+pip install bounter  # install from PyPI
 ```
 
-Then, we've selected a random sample of words as a validation set (unigrams, resp. bigrams) and compared the real
-count of these with the structures.
-All counters use the same validation set.
+Or, if you prefer to install from the [source tar.gz](https://pypi.python.org/pypi/bounter):
+
+```bash
```

Comment:
`tar -xzf  ... ` firstly, or maybe this isn't needed here, `pip install` is enough.